project/NodeJSEnvForcePolyfills.scala

```diff
@@ -5,17 +5,32 @@ import org.scalajs.jsenv.nodejs._
 
 import org.scalajs.core.tools.io._
 
-class NodeJSEnvForcePolyfills(config: NodeJSEnv.Config)
-    extends NodeJSEnv(config) {
-
+final class NodeJSEnvForcePolyfills(config: NodeJSEnv.Config) extends JSEnv {
   def this() = this(NodeJSEnv.Config())
 
-  override protected def vmName: String = "Node.js forcing polyfills"
+  val name: String = "Node.js forcing polyfills"
+
+  private val nodeJSEnv = new NodeJSEnv(config)
+
+  def start(input: Input, runConfig: RunConfig): JSRun =
+    nodeJSEnv.start(patchInput(input), runConfig)
+
+  def startWithCom(input: Input, runConfig: RunConfig,
+      onMessage: String => Unit): JSComRun = {
+    nodeJSEnv.startWithCom(patchInput(input), runConfig, onMessage)
+  }
+
+  private def patchInput(input: Input): Input = input match {
+    case Input.FilesToLoad(fs) => Input.FilesToLoad(forcePolyfills +: fs)
+
+    case _ =>
+      throw new IllegalArgumentException(s"Unsupported input: ${input.getClass}")
+  }
```

Comment:
Should we provide a utility for this in JSEnv? I'm just not sure how to specify it so that we are sure we can always support it for all `Input` types.